site/src/sphinx/advanced-structured-logging.rst

```diff
@@ -61,6 +63,92 @@ What properties can be retrieved?
 
++--------------------------------------------------------------------------------------------------------------+
+| Client connection timing properties                                                                          |
++==========================================+===================================================================+
+| ``dnsResolutionStartTimeMicros``         | when resolving a domain name started, in microseconds since the   |
+|                                          | epoch (01-Jan-1970 00:00:00 UTC)                                  |
++------------------------------------------+-------------------------------------------------------------------+
+| ``dnsResolutionDurationNanos``           | the duration took to resolve a domain name                        |
++------------------------------------------+-------------------------------------------------------------------+
+| ``socketConnectStartTimeMicros``         | when connecting to a remote peer started, in microseconds since   |
+|                                          | the epoch (01-Jan-1970 00:00:00 UTC)                              |
++------------------------------------------+-------------------------------------------------------------------+
+| ``socketConnectDurationNanos``           | the duration took to connect to a remote peer                     |
++------------------------------------------+-------------------------------------------------------------------+
+| ``pendingAcquisitionStartTimeMicros``    | when waiting the completion of an existing connection attempt     |
+|                                          | started, in microseconds since the epoch                          |
+|                                          | (01-Jan-1970 00:00:00 UTC)                                        |
++------------------------------------------+-------------------------------------------------------------------+
+| ``pendingAcquisitionDurationNanos``      | the duration took to wait the completion of an existing connection|
+|                                          | attempt to use one connection in HTTP/2                           |
++------------------------------------------+-------------------------------------------------------------------+
+| ``connectionAcquisitionStartTimeMicros`` | when acquiring a connection started, in microseconds since the    |
+|                                          | epoch (01-Jan-1970 00:00:00 UTC)                                  |
++------------------------------------------+-------------------------------------------------------------------+
+| ``connectionAcquisitionDurationNanos``   | the duration took to get a connection                             |
++------------------------------------------+-------------------------------------------------------------------+
+
+Client connection timing properties
+-----------------------------------
```

Comment:
- How about moving this heading before the table above?
- How about using `^^^^^^^^^` to make this a subsection?